scripts/install/opt/seagate/cortx/motr/bin/motr_mini_prov.py

```diff
@@ -0,0 +1,248 @@
+        time.sleep(SLEEP_SECS)
+        restart_services(["lnet"])
+
+def configure_libfabric(self):
+    pass
+
+def create_lvm(node_name, metadata_dev):
+    try:
+        validate_file(metadata_dev)
+
+        cmd = f"fdisk -l {metadata_dev}"
+        execute_command(cmd, TIMEOUT_SECS)
+
+        cmd = "swapoff -a"
+        execute_command(cmd, TIMEOUT_SECS)
+
+        cmd = f"pvcreate {metadata_dev}"
+        execute_command(cmd, TIMEOUT_SECS)
+
+        cmd = f"vgcreate  vg_metadata_{node_name} {metadata_dev}"
+        execute_command(cmd, TIMEOUT_SECS)
+
+        cmd = f"vgchange --addtag {node_name} vg_metadata_{node_name}"
+        execute_command(cmd, TIMEOUT_SECS)
+
+        cmd = "vgscan --cache"
+        execute_command(cmd, TIMEOUT_SECS)
+
+        cmd = f"lvcreate -n lv_main_swap vg_metadata_{node_name} -l 51%VG"
+        execute_command(cmd, TIMEOUT_SECS)
+
+        cmd = f"lvcreate -n lv_raw_metadata vg_metadata_{node_name} -l 100%FREE"
+        execute_command(cmd, TIMEOUT_SECS)
+
+        cmd = f"mkswap -f /dev/vg_metadata_{node_name}/lv_main_swap"
+        execute_command(cmd, TIMEOUT_SECS)
+
+        cmd = f"test -e /dev/vg_metadata_{node_name}/lv_main_swap"
+        execute_command(cmd, TIMEOUT_SECS)
+
+        cmd = f"swapon /dev/vg_metadata_{node_name}/lv_main_swap"
+        execute_command(cmd, TIMEOUT_SECS)
+
+        cmd = (
+           f"echo \"/dev/vg_metadata_{node_name}/lv_main_swap    swap    "
+           f"swap    defaults        0 0\" >> /etc/fstab"
+        )
+        execute_command(cmd, TIMEOUT_SECS)
+    except:
+        pass
+
+def config_lvm(self):
+    metadata_device = Conf.get(self._index,
+               f'cluster>{self._server_id}')['storage']['metadata_devices']
+    sys.stdout.write(f"[INFO] server_id={self._server_id} "
+                     f" metadata_device={metadata_device[0]}\n")
+    create_lvm(self._server_id, metadata_device[0])
+
+def get_lnet_xface() -> str:
+    lnet_xface = None
+    try:
+        with open(LNET_CONF_FILE, 'r') as f:
+            # Obtain interface name
+            for line in f.readlines():
+                if len(line.strip()) <= 0: continue
+                tokens = re.split(r'\W+', line)
+                if len(tokens) > 4:
+                    lnet_xface = tokens[4]
+                    break
+    except:
+        pass
+
+    if lnet_xface == None:
+        raise MotrError(errno.EINVAL, "Cant obtain iface details from %s"
+                        , LNET_CONF_FILE)
+    if lnet_xface not in os.listdir(SYS_CLASS_NET_DIR):
+        raise MotrError(errno.EINVAL, "Invalid iface %s in lnet.conf"
+                        , lnet_xface)
+
+    return lnet_xface
+
+def check_pkgs(src_pkgs, dest_pkgs):
+    missing_pkgs = []
+    for src_pkg in src_pkgs:
+        found = False
+        for dest_pkg in dest_pkgs:
+            if src_pkg in dest_pkg:
+                found = True
+                break
+        if found == False:
```

Comment:
s/if found == False:/if ! found:/

Like that in C code, we don't use:
found == False 
or
found == true